tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/action/InitBundleTaskAction.java

```diff
@@ -43,9 +45,19 @@ protected void afterTask()
             if( bundlesLocation.toFile().exists() )
             {
                 ServerUtil.addPortalRuntimeAndServer( serverName, bundlesLocation, new NullProgressMonitor() );
+
+                IProject pluginsSDK = CoreUtil.getProject( "plugins-sdk" );
```

Comment:
They could have configured the plugins-sdk directory to be custom. Can we add the properties lookup here?